src/main/java/org/opentest4j/MultipleFailuresError.java

```diff
@@ -90,11 +92,13 @@ private static String pluralize(int count, String singular, String plural) {
 		return count == 1 ? singular : plural;
 	}
 
-	private static String nullSafeMessage(AssertionError failure) {
+	private static String nullSafeMessage(Throwable failure) {
+		if (failure == null) {
```

Comment:
Oh darn. :<

In that case, I'll just reiterate that my personal preference is `NullPointerException`, but if people don't like it, then I won't push against using `IllegalArgumentException` instead. :)